### ISSUE TITLE
When parsing int values in JSON, the resulting csv has them as double

### ISSUE THREAD

**Issue by jslong1** [opendevl/Json2Flat#18]
Take the JSON in the Readme.md for example:
...
{
    "name":"dasd",
    "category": "reference",
    "author": "Nigel Rees",
    "title": "Sayings of the Century",
    "price": 8.95,
    "marks" : [3,99,89]
}
...

When the result for "marks", which is originally of type int, is parsed and examined in a text editor, it looks like this:
3.0,99.0,89.0

Can there be a fix for this, or are all ints going to be defaulted to doubles?

**Comment by curious95** [comment]
Yeah this issue is persisting. it will be fixed in the next build.

**Comment by jslong1** [comment]
That's great. Any ETA on when the next build is going to be released?

**Comment by curious95** [comment]
@jslong1 it'll take 1 more week.

**Comment by masudsarkar** [comment]
@curious95 was this fixed! i could still see the issue persisting.

**Comment by avalakusuma** [comment]
@curious95,
I can still see the issue in the 1.0.3 release. For my case it is converting id fields in my json file to double values in csv. Could you please fix the issue or is there any work around for this?

**Comment by AbhirajRTBA** [comment]
This issue still exist .Could you please fix the issue or is there any work around for this?
